server/channel_npc.go

```diff
@@ -228,11 +228,9 @@ func (server *ChannelServer) npcShop(conn mnet.Client, reader mpacket.Reader) {
 			return
 		}
 
-		_, err = plr.TakeItem(itemID, amount)
+		invID := byte(2) // Need to adjust this to find the inventory ID for selling
```

Comment:
First digit is the inventory id: 1 = equip, 2 = use, 3 = set-up, 4 = etc, all others = cash

Divide the item id by 1000000 to get the value

Reply:
Oh perfect. I was not aware of that structure. I will update it to find the inventory ID that way.